src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrCodeflower.cs

```diff
@@ -270,9 +269,19 @@ protected abstract record Codeflow(string SourceSha, string TargetSha)
         public abstract string RepoSha { get; init; }
 
         public abstract string VmrSha { get; init; }
+
+        public string GetBranchName() => $"codeflow/{Name}/{Commit.GetShortSha(SourceSha)}-{Commit.GetShortSha(TargetSha)}";
+
+        public abstract string Name { get; }
```

Comment:
Consider: use Enum for this to make clear what the possible values are.

Reply:
I introduced this so that we can avoid the `flow is BackFlow ? "backward" : "forward"` in quite a few places. The enum would have to be named lowercased or we would have to keep lowercasing it. Is it worth it?